### PR TITLE
Add example of Apache Ignite config

### PR DESCRIPTION
**Pull request by char16t**

### CODE REVIEW

File: example_configs/ignite.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+# This is example of getting Apache Ignite cache metrics.
+#
+# This metrics will be available after enabling JMX metrics gathering
+# Instructions available on https://apacheignite.readme.io/v2.0/docs/memory-and-cache-metrics
+# or by search query "ignite jmx metrics"
+#
+# 1. Enable statistic for your cache (example for Scala)
```
Comment (brian-brazil) [inline example_configs/ignite.yml]: If you have access to change the code, then using client_java directly is preferred to using JMX.

Comment (char16t) [reply example_configs/ignite.yml]: Complex question. Metrics registration needs to be explicitly call. Some metrics are not directly dependent on application actions. For Example, "EntryProcessorHits", "KeysToRebalanceLeft". I think that it is undesirable to register these metrics yourself

Comment (char16t) [reply example_configs/ignite.yml]: Part of available Ignite metrics: screenshot-1

Comment (brian-brazil) [reply example_configs/ignite.yml]: The code should be registering the metrics, and storing them as static class methods.

**Pull request closed without merging**